gpii/node_modules/flowManager/configs/development.json

```diff
@@ -25,8 +25,17 @@
             "flat": {
                 "url": "http://localhost:8081"
             },
```

Comment:
Is there a reason why you've modified this file as opposed to the review4 scenario 3 file: https://github.com/AndreasStiegler/universal/blob/review4/gpii/configs/review4.scenario3.production.json#L19-L31 ? If not, you should edit that file with the new SMM URLs instead and leave this one as is